### ISSUE TITLE
My Argocd Failed to synchronize application status. "Failed to load target state: failed to generate manifest for source 1 of 1: rpc error: code = Unknown desc = unknown error: remote: Internal API unreachable"

### ISSUE THREAD

**Issue by pantao24** [argoproj/argo-cd#17609]
### Discussed in https://github.com/argoproj/argo-cd/discussions/17608

<div type='discussions-op-text'>

<sup>Originally posted by **pantao24** March 25, 2024</sup>
![image-20240321203157802](https://github.com/argoproj/argo-cd/assets/40918197/bcf1c2e6-4598-4adc-b096-11c6c25a9a28)
</div>

**Comment by Killpit** [comment]
Did you use any CI/CD pipeline when you use ArgoCD?

**Comment by pantao24** [comment]
> 您在使用ArgoCD时是否使用过任何CI/CD管道？

使用Jenkins进行构建，配置文件存储在自建的Gitlab中，Gitlab的性能不足，Argocd会报如上错误么

**Comment by pantao24** [comment]
> 您在使用ArgoCD时是否使用过任何CI/CD管道？

Using Jenkins to build, the configuration file is stored in the self-built Gitlab, Gitlab performance is insufficient, Argocd will report such errors

**Comment by Killpit** [comment]
Did you check connectivity using `kubectl` and made sure it's connected to API `kubectl cluster-info`? Because ArgoCD requires you to use Kubernetes and it's a good practice to check connectivity beforehand

**Comment by pantao24** [comment]
> Did you check connectivity using `kubectl` and made sure it's connected to API `kubectl cluster-info`? Because ArgoCD requires you to use Kubernetes and it's a good practice to check connectivity beforehand

Yes, I have checked the connectivity with the Kubernetes cluster

**Comment by Killpit** [comment]
So what do they say and can you give the outputs?
